# Lab 11 - Group - Midterm 2 Review - [5 points] - Solutions



## <u>Case Study</u>: Inference on a Population Proportion

Suppose that a Calculus student is curious about the proportion of former Calculus students that got A's in the class that they are currently enrolled in (taught by Dr. Difficult). This class has had thousands of students enrolled in the past. The student collects a random sample of 40 former students who have taken the class and asks them if they got an A in the class (or not). 60% of the students in the sample got an A.

This student also knows that the proportion of ALL students who took another Calculus class (taught by Dr. Eezee) and got an A was 0.80. The student would like to determine if there is evidence to suggest that the proportion of ALL students who have taken Dr. Difficult's class that get A's is *different* than the the proportion of ALL students who have taken Dr. Eezee's class that get A's.

So we will be testing the following hypotheses below.

$H_0: p=0.8$

$H_A: p\neq 0.8$,

where $p=$ the proportion of all student's who have taken Dr. Difficult's class that got A's.

In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt

from scipy.stats import bernoulli
from scipy.stats import binom
from scipy.stats import norm
from scipy import stats
from scipy.stats import t

import statsmodels.api as sm
import statsmodels.formula.api as smf

import seaborn as sns; sns.set()
from scipy.stats import f

import math

### Test Statistic

First, we calculate the test statistic for this hypothesis test.

In [2]:
n = 40

#Test statitsic pieces
point_estimate = 0.6
null_value = 0.8
standard_error = np.sqrt(null_value*(1-null_value)/n)

#Test statistic
test_stat = (point_estimate - null_value)/standard_error
test_stat

-3.1622776601683804

### p-value 

Then, we calculate the p-value for this hypothesis test.

In [3]:
from scipy.stats import norm
pvalue = 2*(1-norm.cdf(np.abs(test_stat)))
pvalue

0.0015654022580025018

###  Fill in the Blank Section [0.25 points for each number or letter blank]

Write your answers below to the corresponding numbered (or lettered) blanks in the attached pdf in this folder.

1. sample proportions 
2. without 
3. sample proportion
4. 0 
5. 1
6. CLT
7. normal 
a. the sample is random
b. n = 40 < all thoasands of students who were previously employeed. 
c. 40(0.8) = 32 and 40(0.2) = 8
8. 3.1622776601683804
9. alternative hypothesis is true
10. 3.1622776601683804
11. 0.8
12. normal
13. 0
14. 1
15. 0
16. is
17. does

